README.md

```diff
@@ -186,6 +176,47 @@ qemu-system-x86_64 \
     -initrd /tmp/initramfs.linux_amd64.cpio
 ```
 
+## SystemBoot
+
+SystemBoot is a set of bootloaders written in Go. It is meant to be a
+distribution for LinuxBoot to create a system firmware + bootloader. All of
+these use `kexec` to boot. The commands are in [cmds/boot](cmds/boot).
+
+*   `pxeboot`: a network boot client that uses DHCP and HTTP or TFTP to get a
+    boot configuration which can be parsed as PXELinux or iPXE configuration
+    files to get a boot program.
+
+*   `fbnetboot`: a network boot client that uses DHCP and HTTP to get a boot
+    program based on Linux, and boots it. To be merged with `pxeboot`.
+
+*   `localboot`: a tool that finds bootable kernel configurations on the local
+    disks and boots them.
+
+*   `boot2`: similar to `localboot`, finds a bootable kernel configuration on
+    disk (GRUB or syslinux) and boots it. To be merged into `localboot`.
+
+*   `systemboot`: a wrapper around `fbnetboot` and `localboot` that just mimicks
+    a BIOS/UEFI BDS behaviour, by looping between network booting and local
+    booting. Use `-uinitcmd` argument to the u-root build tool to make it the
+    boot program.
+
+This project started as a loose collection of programs in u-root by various
+LinuxBoot contributors, as well as a personal experiment by
+[Andrea Barberio](https://github.com/insomniacslk) that has since been merged
+in. It is now an effort of a broader community and graduated to a real project
+for system firmwares.
+
+More detailed information about the build process for a full LinuxBoot firmware
+image using u-root/systemboot and coreboot can be found in the
+[LinuxBoot book](https://github.com/linuxboot/book) chapter 10,
```

Comment:
The chapter numbers got reorders. Perhaps just link directly to https://github.com/linuxboot/book/tree/master/coreboot.u-root.systemboot